src/test/java/tests/FileUploaderTest.java

```diff
@@ -0,0 +1,31 @@
+package tests;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class FileUploaderTest extends BaseTest {
+
+    /* 1. Загрузить файл
+       2. Проверить, что имя файла на странице совпадает с именем загруженного файла */
+
+    private static final String EXPECTED_CONFIRMATION_MESSAGE = "File Uploaded!";
+    private static final String EXPECTED_ERROR = "Internal Server Error";
+    private static final String PATH_TO_FILE = "C:\\hello.png";
```

Comment:
better to place that file in Framework and specify the relative path to it. Because this test will be failed on my side